components/icons/future_threshold.tsx

```diff
@@ -0,0 +1,74 @@
+      fill="#E0E0E0"
+    />
+    <path
+      d="M73 5C67.9282 5 63.8011 5 59.7238 5C55.6464 5 56.0689 5 51.4199 5C50.2247 5 46 5 46 5"
+      stroke="#E0E0E0"
+      stroke-width="2"
+      stroke-dasharray="2 2"
```

Comment:
For some reason React is giving me warnings on the server unless these are changed as so:
```suggestion
      strokeWidth="2"
      strokeDasharray="2 2"
```